Reconstruction/RecCalorimeter/src/components/CreatePositionedHit.cpp

```diff
@@ -0,0 +1,121 @@
+#include "CreatePositionedHit.h"
```

Comment:
This is not really reconstruction related, is it? Could go into the simulation package.